test/hotspot/jtreg/vmTestbase/nsk/share/Log.java

```diff
@@ -220,10 +218,12 @@ public boolean verbose() {
      * Enable or disable verbose mode for printing messages.
      */
     public void enableVerbose(boolean enable) {
+        if (!enable) {
+            throw new RuntimeException("The non-verbose logging is not supported.");
+        }
         if (!verbose) {
             flushLogBuffer();
         }
```

Comment:
flushLogBuffer() is never executed. I think you can remove it.

Reply:
fixed